src/web/app/pages-instructor/student-list/__snapshots__/student-list.component.spec.ts.snap

```diff
@@ -0,0 +1,3044 @@
+            </a>
+            
+            
+            
+            
+            
+            
+            <button
+              class="btn btn-light btn-sm mr-2"
+              ng-reflect-klass="btn btn-light btn-sm mr-2"
+              ng-reflect-ng-class="[object Object]"
+              ng-reflect-ngb-tooltip="Delete the student and the cor"
+            >
+              Delete
+            </button>
+            
+            
+            <a
+              class="btn btn-light btn-sm mr-2"
+              href="/web/instructor/students/records?courseid=&studentemail=danny.e.tmms@gmail.tmt"
+              ng-reflect-klass="btn btn-light btn-sm mr-2"
+              ng-reflect-ng-class="[object Object]"
+              ng-reflect-ngb-tooltip="View all data about this stude"
+              ng-reflect-query-params="[object Object]"
+              ng-reflect-router-link="/web/instructor/students/recor"
+            >
+               All Records
+            </a>
+            
+          </td>
+        </tr>
+        <tr
+          s=""
+        >
+          <td
+            class="align-center"
+          >
+            
+            
+            <a
+              href="null"
+              role="button"
+            >
+              View Photo
+            </a>
+            
+          </td>
+          
+          <td>
+            Tutorial Group 1
+          </td>
+          <td>
+            Team 1
+          </td>
+          <td>
+            Emma Farrell
+          </td>
+          <td
+            class="align-center"
+          >
+            Yet to join
+          </td>
+          <td>
+            emma.f.tmms@gmail.tmt
+          </td>
+          <td
+            class="no-print align-center"
+          >
+            
+            
+            <a
+              class="btn btn-light btn-sm mr-2"
+              href="/web/instructor/courses/student/details?courseid=&studentemail=emma.f.tmms@gmail.tmt"
+              ng-reflect-klass="btn btn-light btn-sm mr-2"
+              ng-reflect-ng-class="[object Object]"
+              ng-reflect-ngb-tooltip="View the details of the studen"
+              ng-reflect-query-params="[object Object]"
+              ng-reflect-router-link="/web/instructor/courses/studen"
+            >
+               View
+            </a>
+            
+            
+            <a
+              class="btn btn-light btn-sm mr-2"
+              href="/web/instructor/courses/student/edit?courseid=&studentemail=emma.f.tmms@gmail.tmt"
+              ng-reflect-klass="btn btn-light btn-sm mr-2"
+              ng-reflect-ng-class="[object Object]"
+              ng-reflect-ngb-tooltip="Use this to edit the details o"
+              ng-reflect-query-params="[object Object]"
+              ng-reflect-router-link="/web/instructor/courses/studen"
+            >
+               Edit
+            </a>
+            
+            
+            
+            
+            
+            
+            <button
```

Comment:
Er should this be displayed if enableRemindButton is false?

Reply:
No it should not. You are looking at the test case where `enableRemindButton` is set to true....

Reply:
Whoops, my bad